whylabs_toolkit/monitor/diagnoser/recommendation/recommended_change.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import annotations
+from typing import Optional, List
+
+from whylabs_toolkit.monitor.models import Analyzer
+
+from whylabs_toolkit.monitor.diagnoser.models import ConditionRecord
+from whylabs_toolkit.monitor.diagnoser.helpers.describe import describe_truncated_list
+
+
+class RecommendedChange:
+    name = ""
+    summary = ""
+    manual = True
+    required_info: List[str] = []
+
+    @classmethod
+    def from_condition(cls, condition: ConditionRecord) -> RecommendedChange:
+        return cls(condition.columns if condition.columns is not None else [], condition.info)
+
+    def __init__(self, columns: List[str], info: Optional[dict] = None):
+        self.columns = columns
+        self.info = info
+
+    def merge(self, change: RecommendedChange) -> RecommendedChange:
+        if change.name != self.name:
```

Comment:
seems that (at least for this class) `self.name` is never reassigned

Reply:
this is guarding against being asked to merge in a recommended change of a different type, so it can happen